clients/nodejs/cac-client/client.ts

```diff
@@ -0,0 +1,106 @@
+import * as ffi from 'ffi-napi';
+import * as ref from 'ref-napi';
+import * as path from 'path';
+import os from 'os';
+
+let platform = os.platform();
+
+let file =
+    platform == "darwin" ?
+    '../../../target/debug/libcac_client.dylib' :
```

Comment:
Are we sure that file will be present at this path in case of release build too (its specifically pointing to debug directory)?